### PR TITLE
add pound-force inch torque conversion

### PR DESCRIPTION
**Pull request by adrianinsaval**
I guess this would be a first step towards https://github.com/ferraridamiano/ConverterNOW/issues/251

### CODE REVIEW

File: lib/properties/torque.dart [hunk 1]
```diff
@@ -42,6 +44,10 @@ class Torque extends DoubleProperty<TORQUE> {
               coefficientProduct: 1.35581794902490555,
               name: TORQUE.poundForceFeet,
             ),
+            ConversionNode(
+              coefficientProduct: 0.1129848290854088,
+              name: TORQUE.poundForceInch,
+            ),
```
Comment (ferraridamiano) [inline lib/properties/torque.dart:50]: I suggest to implement it as 1/12 of a pound force feet

File: test/getters_test.dart [hunk 2]
```diff
@@ -466,7 +466,9 @@ void main() {
     runGetterTest(property.newtonMeter, property.getUnit(TORQUE.newtonMeter));
     runGetterTest(property.dyneMeter, property.getUnit(TORQUE.dyneMeter));
     runGetterTest(
-        property.poundForceFeet, property.getUnit(TORQUE.poundForceFeet));
+        property.poundForceFeet, property.getUnit(TORQUE.poundForceFeet));    
+    runGetterTest(
+        property.poundForceFeet, property.getUnit(TORQUE.poundForceInch));
```
Comment (ferraridamiano) [inline test/getters_test.dart:471]: Here there is a copy/paste error

**Review by ferraridamiano — COMMENTED** [review]
Thank you for this PR! Let's just fix a couple of issue and make the CI happy and then we are ready to merge

File: lib/properties/torque.dart (hunk 1 above)
Comment (adrianinsaval) [reply lib/properties/torque.dart:50]: to be honest I am not familiar with the laguage, do you have a suggestion on how to reference the value or should I just write `1.35581794902490555/12`

File: test/getters_test.dart (hunk 2 above)
Comment (adrianinsaval) [reply test/getters_test.dart]: ups!

File: lib/properties/torque.dart (hunk 1 above)
Comment (ferraridamiano) [reply lib/properties/torque.dart:50]: Add the `ConversionNode` as a List to the `leafNode` property of the pound force feet `conversionNode`

**Comment by ferraridamiano** [comment]
There seems to be also format issues. Could you please run `dart format .`?

**Comment by ferraridamiano** [comment]
Closing this for now since for lack of activity in the past year. Feel free to reopen it or just ping me

**Pull request closed without merging**